src/generate-cli-options.ts

```diff
@@ -0,0 +1,144 @@
+import assert from "node:assert";
+import { readFileSync, realpathSync, rmSync, writeFileSync } from "node:fs";
+import { basename, dirname, join } from "node:path";
+import type { AstroIntegration } from "astro";
+import { LYCHEE_VERSION } from "./lychee-version";
+
+// https://raw.githubusercontent.com/lycheeverse/lychee/master/README.md
+const url = `https://raw.githubusercontent.com/lycheeverse/lychee/refs/tags/${LYCHEE_VERSION}/README.md`;
+
+const TEMPLATE = "README-OPTIONS-PLACEHOLDER";
+
+function extractHelpFromReadme(readme: string) {
+	const [, section] = readme.split(/### Commandline Parameters/, 2);
+	if (!section)
+		throw new Error(
+			"LycheeCliOptions: commandline parameters section not found in readme",
+		);
+
+	const [, text] = section.split("\n```help-message\n", 2);
+	if (!text)
+		throw new Error(
+			"LycheeCliOptions: ```help-message marker not found in commandline parameters section",
+		);
+
+	const [helpText] = text.split("\n```\n", 2);
+	if (!helpText)
+		throw new Error(
+			"LycheeCliOptions: closing ``` marker not found after ```text",
+		);
+
+	return helpText;
```

Comment:
```suggestion
	return helpText.trim();
```

To avoid future issues with whitespace